src/App.js

```diff
@@ -17,7 +16,6 @@ const App = () => (
         <Route exact path="/bookstore" element={<Books />} />
```

Comment:
## OPTIONAL
- [ ] It would be practical if we keep the main page as the main just letting the path as ' / ' instead of ' /bookstore ' to be able to see first the books component rendered.